Mathlib/CategoryTheory/Adjunction/Unique.lean

```diff
@@ -3,30 +3,21 @@ Copyright (c) 2020 Bhavik Mehta. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Bhavik Mehta, Thomas Read, Andrew Yang, Dagur Asgeirsson, Joël Riou
 -/
-import Mathlib.CategoryTheory.Adjunction.Basic
+import Mathlib.CategoryTheory.Adjunction.Mates
 /-!
 
 # Uniqueness of adjoints
 
 This file shows that adjoints are unique up to natural isomorphism.
 
 ## Main results
```

Comment:
I think the cause of the duplication in the first place was poor documentation somewhere - in particular it wasn't clear enough that these definitions already existed (and there's a good chance that's my fault!). Do you have an idea where we should add cross-references to ensure others can't make the same mistake you did?

Reply:
The way this happened was that previously, uniqueness of adjoints lived in the file `Adjunction.Opposites` and was a different construction. I felt that this was a more natural construction and changed it, creating the file `Adjunction.Unique`, without knowing about the material in `Adjunction.Mates`.

I guess it might help to have some signposting in the module docstring `Adjunction.Basic`. Currently, the docstring of the definition of `Adjunction` points to the file `Adjunction.Unique` for uniqueness of adjoints (I guess that's because previously it pointed to `Adjunction.Opposites`, which is hard to guess. But we might want more like that in `Adjunction.Basic`

Reply:
I've added some documentation to `Adjunction.Basic`. It's basically saying one line about what's in each file in the `Adjunction` directory, and feels a bit obvious at times. What do you think @b-mehta?

Reply:
It looks excellent! Thanks for adding this.